dom0/fpf-apt-test-repo.sls

```diff
@@ -1,19 +1,8 @@
 # -*- coding: utf-8 -*-
 # vim: set syntax=yaml ts=2 sw=2 sts=2 et :
-
-# Handle misconfigured jessie-backports repo in default debian-9 TemplateVM.
-# The Jessie repos aren't maintained anymore, and their inclusion causes
-# even apt update to fail.
-remove-jessie-backports-repo:
-  file.line:
-    - name: /etc/apt/sources.list
-    # Unclear why "Delete" *must* be capitalized, but that's the case!
-    - mode: delete
-    - match: jessie-backports
-    # quiet param seems to be ignored, so using "onlyif" to test existence
-    - quiet: True
-    - onlyif:
-      - test -f /etc/apt/sources.list
+#
+include:
+  - update.qubes-vm
```

Comment:
Where is this salt file located? Is this provided by qubes' build in salt mgmt commands? Might be worth adding a comment here for future maintainers.

Reply:
+1 for more comments everywhere, will add.